config/locales/ar.yml

```diff
@@ -496,6 +496,10 @@ ar:
     world_locations:
       all: كافة المواقع
       label: مواقع في العالم
+  email_signups:
+    new:
+      description:
+      title:
```

Comment:
Wow, this is a beast of a step.

Reply:
The ordering here is different from the other files. It may make more sense for people in the future if they are consistent.

Reply:
I think that's a minor concern - I doubt it will cause an issue, because we don't rely on the ordering for anything. Whitehall has significant issues with its use of locales, but we're not going to fix any of them here, unfortunately.